model/project_ref.go

```diff
@@ -972,6 +975,28 @@ func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) error {
 	return nil
 }
 
+func (p *ProjectRef) AuthorizedForGitTag(ctx context.Context, user string, token string) (bool, error) {
+	if utility.StringSliceContains(p.GitTagAuthorizedUsers, user) {
+		return true, nil
+	}
+	// check authorized teams
+	if token == "" {
```

Comment:
Yeah I already thought of the alternatives. I liked having a second check for token here, in case the settings for the rest route were outdated. I don't mind just moving this into the repotracker though.